static/js/components/forms/ProfileFormFields.js

```diff
@@ -0,0 +1,499 @@
+export const ProfileFields = () => (
+  <React.Fragment>
+    <div className="form-group">
+      <div className="row">
+        <div className="col">
+          <label htmlFor="profile.gender" className="font-weight-bold">
+            Gender*
+          </label>
+
+          <Field
+            component="select"
+            name="profile.gender"
+            className="form-control"
+          >
+            <option value="">-----</option>
+            <option value="f">Female</option>
+            <option value="m">Male</option>
+            <option value="o">Other / Prefer not to say</option>
+          </Field>
+          <ErrorMessage name="profile.gender" component={FormError} />
+        </div>
+        <div className="col">
+          <label htmlFor="profile.birth_year" className="font-weight-bold">
+            Year of Birth*
+          </label>
+          <Field
+            component="select"
+            name="profile.birth_year"
+            className="form-control"
+          >
+            <option value="">-----</option>
+            {reverse(range(seedYear - 120, seedYear - 14)).map((year, i) => (
+              <option key={i} value={year}>
+                {year}
+              </option>
+            ))}
+          </Field>
+          <ErrorMessage name="profile.birth_year" component={FormError} />
+        </div>
+      </div>
+    </div>
+    <div className="form-group">
+      <label htmlFor="profile.company" className="font-weight-bold">
+        Company*
+      </label>
+      <Field type="text" name="profile.company" className="form-control" />
+      <ErrorMessage name="profile.company" component={FormError} />
+    </div>
+    <div className="form-group">
+      <label htmlFor="profile.job_title" className="font-weight-bold">
+        Job Title*
+      </label>
+      <Field type="text" name="profile.job_title" className="form-control" />
+      <ErrorMessage name="profile.job_title" component={FormError} />
+    </div>
+    <div className="form-group dotted" />
+    <div className="form-group">
+      <label htmlFor="profile.industry" className="font-weight-bold">
+        Industry
+      </label>
+      <Field
+        component="select"
+        name="profile.industry"
+        className="form-control"
+      >
+        <option value="">-----</option>
+        {EMPLOYMENT_INDUSTRY.map((industry, i) => (
+          <option key={i} value={industry}>
+            {industry}
+          </option>
+        ))}
+      </Field>
+    </div>
+    <div className="form-group">
+      <label htmlFor="profile.job_function" className="font-weight-bold">
+        Job Function
+      </label>
+      <Field
+        component="select"
+        name="profile.job_function"
+        className="form-control"
+      >
+        <option value="">-----</option>
+        {EMPLOYMENT_FUNCTION.map((jobFunction, i) => (
+          <option key={i} value={jobFunction}>
+            {jobFunction}
+          </option>
+        ))}
+      </Field>
```

Comment:
we have a few `selects` which look pretty similar, would it be worthwhile to add a component like this?

```js
function SelectField(props) {
  const { name, label, options } = props

  return (
    <>
      <label htmlFor={name} className="font-weight-bold">
        { label }
      </label>
      <Field
        component="select"
        name={name}
        className="form-control"
      >
        <option value="">-----</option>
        {options.map(([value, label], i) => (
          <option key={i} value={value}>
            {label}
          </option>
        ))}
      </Field>
    </>
  )
}
```